qxf2/qxf2-survey
# Added the Neo4j Integration with FastAPI

## Description

Adding Neo4j Integration to FastAPI. This would allow the Data to be queried from a Graph Database and setup a Database Connection which can be tested and verified from an API Endpoint.

## Issue Reference

The Jira Ticket is available [here](https://qxf2services.atlassian.net/jira/software/projects/QELO/boards/32?selectedIssue=QELO-18)

## Testing

Steps to run the Code:

- Make a new Project on the Neo4j Desktop. Create a Database on it.

- Specify the password for the Database. The user would be the default `neo4j`

- Install all the Packages and run the Database.

- Run the command: `uvicorn main:app --reload`

- Go to the endpoint:` /graph` and you will see a message showing that the Database Connection is successful.
